### DISCUSSION TITLE
How can I import data from PostgreSQL without passing it to pandas first?

### DISCUSSION THREAD

**Discussion by MarcIrving** [atoti/atoti#588] [Q&A] [answered]
i think i was doing it wrong , i need to get my data from my db and put them in dataframe in pandas then i use atoti to read the tables , 
is there any way to do it directly without this long process ?

**Comment by tibdex** [comment, marked as answer]
[`Session.read_sql()`](https://docs.atoti.io/0.6.5/lib/atoti/atoti.session.html#atoti.Session.read_sql) and [`Table.load_sql()`](https://docs.atoti.io/0.6.5/lib/atoti/atoti.table.html#atoti.Table.load_sql) are made for this.
